meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-common.cmd.in

```diff
@@ -133,6 +133,12 @@ if test "${fiovb.is_secondary_boot}" = "0"; then
 		imx_secondary_boot 0
 		reset
 	fi
+else
+	# Save fiovb.is_secondary_boot state for allowing userspace
```

Comment:
why do we need this in userspace?

Reply:
We don't, I just wanted this to be available to easy testing/validation.

There is currently no way, from userspace, to identify if we booted or not via secondary.